packages/editor-ui/src/constants.ts

```diff
@@ -444,4 +445,11 @@ export enum STORES {
 
 export const EXPRESSION_EDITOR_PARSER_TIMEOUT = 15_000; // ms
 
+export const KEEP_AUTH_IN_NDV_FOR_NODES = [
+	HTTP_REQUEST_NODE_TYPE,
+	WEBHOOK_NODE_TYPE,
+	WAIT_NODE_TYPE,
```

Comment:
Discussed this yesterday with Max and Nik and the agreement is to add wait node as an exception here. This is because it's credentials are optional + auth field is not always visible, depending on the `resume` field.

Reply:
`resume` is an optional field though right? so should not apply anyways

Reply:
That's true, the reason why it wasn't picked up is because current logic is checking only immediate dependencies to determine if a field is mandatory. Will update it to check the whole dependency tree and remove the node from exceptions list.

Reply:
nvm probably fine as is..